package.json

```diff
@@ -31,6 +31,7 @@
     "@turf/turf": "^6.5.0",
     "jquery": "^3.5.1",
     "polygon-clipping": "^0.15.7",
+    "typescript": "^5.6.2",
```

Comment:
id save it under `devDependencies` (npm i typescript --save-dev) since its only used when making the build.

actually now that im thinking about it since we package the whole tool with webpack maybe all these should be `devDependencies`, since a user may only require the `ulabel.js` file to run. not sure on that tho

Reply:
I think `devDependencies` would be the spot for things not needed for the build, like linters/formatters. Once it's built, nothing else is needed

Reply:
anything listed in `dependencies` is installed when the package is installed via `npm i ulabel`. Since all that is really needed is `dist/ulabel.js`, in reality nothing else NEEDS to be installed for the package to work, no?

when someone is developing/building ulabel from source, running `npm install` in the `ulabel` directory will by default install all `dependencies` AND `devDependencies`, so there wouldn't be any issue there.

so by putting everything in `devDependencies` we lose some distinction between what are core to the app's function vs nice for development, but we reduce the number of extraneous dependencies that are installed when using the package. personally i'd lean towards moving everything to `devDependencies`

Reply:
This thread seems to indicate everything into `devDependencies` like you said: https://github.com/webpack/webpack/issues/520

Not relevant for our case with `webpack`, but `npm` has a `bundleDependencies` for this: https://stackoverflow.com/a/55133339